lib/overcommit/hook/pre_push/git_lfs.rb

```diff
@@ -0,0 +1,43 @@
+module Overcommit::Hook::PrePush
+  # Replaces the pre-push hook installed by `git lfs install` that is then moved aside by
+  # `overcommit --install`
+  #
+  # Enable this hook in .overcommit.yml like so:
```

Comment:
No need for this. You should add the default configuration to `config/default.yml`. There are plenty of examples to pull from there.